solr/solrj/src/test/org/apache/solr/client/solrj/impl/CloudSolrClientBuilderTest.java

```diff
@@ -63,22 +66,20 @@ public void testSeveralZkHostsSpecifiedSingly() throws IOException {
   }
   
   @Test
-  public void testSeveralZkHostsSpecifiedTogether() throws IOException {
```

Comment:
This test was exactly the same as the one above, just with differently formatted newlines.